src/pages/RegisterFamilyUsers.vue

```diff
@@ -40,7 +40,7 @@
         </div>
       </header>
       <div>
-        <div v-if="activeIndex === 1">
+        <div v-if="activeIndex === 1 && spinner !== true">
```

Comment:
Minor suggestion,  `spinner !== true` can be simplified to `!spinner` for better readability
 
```suggestion
        <div v-if="activeIndex === 1 && !spinner">
```

Reply:
However, in terms of UX, it would be better to disable the button instead of removing to avoid layout shifts

Reply:
Perhaps we could use the `loading` and `loadingIcon` property of `PvButton` (see https://primevue.org/button/#api.button.props.loading).